store/tikv/kv.go

```diff
@@ -198,6 +208,39 @@ func (s *KVStore) BeginWithExactStaleness(txnScope string, prevSec uint64) (*KVT
 	return txn, nil
 }
 
+// BeginWithMinStartTS begins transaction with the least startTS
+func (s *KVStore) BeginWithMinStartTS(txnScope string, minStartTS uint64) (*KVTxn, error) {
+	minSafeTS := uint64(0)
+	if txnScope == oracle.GlobalTxnScope {
+		minSafeTS = s.getGlobalMinSafeTS()
```

Comment:
I think `getGlobalMinSafeTS` and `getMinSafeTSByStores` can be merged into one? You could just pass `s.regionCache.getStoresByType(tikvrpc.TiKV)` to `getMinSafeTSByStores`.

Reply:
`getGlobalMinSafeTS` removed